xlators/storage/posix/src/posix-aio.c

```diff
@@ -233,11 +256,7 @@ posix_aio_readv(call_frame_t *frame, xlator_t *this, fd_t *fd, size_t size,
     if (iobuf)
         iobuf_unref(iobuf);
```

Comment:
Can't this be moved to posix_aio_cb_fini() too since we capture it in paiocb->iobuf?  Likewise in posix_aio_readv_complete() too